.github/workflows/ci-core.yml

```diff
@@ -68,14 +69,18 @@ jobs:
           # non-integration-tests - only changes made outside of the `integration-tests` directory
           # non-ignored - only changes except for the negated ones
           #  - This is opt-in on purpose. To be safe, new files are assumed to have an affect on CI Core unless listed here specifically.
+          # Enable listing of files matching each filter.
+          # Paths to files will be available in `${FILTER_NAME}_files` output variable.
+          # Paths will be formatted as JSON array
+          list-files: json
           filters: |
             non-integration-tests:
               - '**'
               - '!integration-tests/**'
             non-ignored:
               - '**'
               - '!docs/**'
-              - '!integration-tests/**'
+              - '!fuzz/**'
```

Comment:
Why was this changed? This is a negation so it ignores the changes for the paths with a `!`. If `fuzz/**` is changed it should run CI-Core, this adds it to the ignore kind of breaking that logic.

If `integration-tests` is changed we don't need to run ci-core because that's not part of the core module or deployments module. That is for the integration tests ran in the [integration tests workflow](https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/integration-tests.yml).

What this change allows is for someone to change the `fuzz/fuzz_all_native.py` script and then the fuzz tests will be skipped.